Add ports to allow incoming traffic (#2680)
theforeman/foreman-documentation: guides/common/modules/ref_ports-and-firewall-requirements.adoc

| 67 | UDP | DHCP | Client | Dynamic IP | DHCP (optional)
| 69 | UDP | TFTP | Client | TFTP Server (optional) |
| 443 | TCP | HTTPS | {SmartProxy} | {ProjectName} API | Communication from {SmartProxy}

> | 443, 80 | TCP | HTTPS, HTTP| Client | Global Registration | Registering hosts to {Project}

Reviewer: ```suggestion
| 443, 80 | TCP | HTTPS, HTTP | Client | Global Registration | Registering hosts to {Project}
```

| 69 | UDP | TFTP | Client | TFTP Server (optional) |
| 443 | TCP | HTTPS | {SmartProxy} | {ProjectName} API | Communication from {SmartProxy}
ifdef::katello,satellite,orcharhino[]
| 443 | TCP | HTTPS | Client | Content Host registration | Initiation

Uploading facts

> Sending installed packages and traces

Reviewer: Let's remove this, because we don't know for sure for which registration method it's supposed to be (at least I don't know)

Author: These processes take place regardless of any method and via `443` only. Can we replace it with simple `Host Registration` to avoid confusion?

| 443 | TCP | HTTPS | {ProjectName} | Content Mirroring | Management
| 443 | TCP | HTTPS | {ProjectName} | {SmartProxy} API | Smart Proxy functionality
| 443, 80 | TCP | HTTPS, HTTP | {SmartProxy} | Content Retrieval | Content
| 443, 80 | TCP | HTTPS, HTTP | Client | Content Retrieval | Content

> | 443, 80 | TCP | HTTPS, HTTP| Client | Content Host Registration | Katello CA Consumer RPM

Reviewer: Let's remove this, because this method is deprecated anyway.

> | 443, 80 | TCP | HTTPS, HTTP| Client | Global Registration | Allow incoming traffic to {Project} for host registration

Reviewer: ```suggestion
| 443, 80 | TCP | HTTPS, HTTP| Client | Global Registration | Registration of hosts to {Project}
```

Reviewer 2: Gobal registration is not a Katello-only feature, but in core Foreman.

Author: Addressed both :+1:
